picard/config_upgrade.py

```diff
@@ -199,4 +209,5 @@ def upgrade_config():
     cfg.register_upgrade_hook(upgrade_to_v1_4_0_dev_3)
     cfg.register_upgrade_hook(upgrade_to_v1_4_0_dev_4)
     cfg.register_upgrade_hook(upgrade_to_v1_4_0_dev_5)
+    cfg.register_upgrade_hook(upgrade_to_v1_4_0_dev_6)
```

Comment:
For this to work, you'll need to update version in https://github.com/metabrainz/picard/blob/master/picard/__init__.py#L25
Better do it at the same time, and test if upgrade hook works:

1. back up your current config file (1.4dev5)
2. run patched Picard (1.4dev6)
3. ensure old options are converted to new ones and new config file is saved